### DISCUSSION TITLE
Lesson 10: Aave syntax question

### DISCUSSION THREAD

**Discussion by toddabraham** [smartcontractkit/full-blockchain-solidity-course-py#109] [Q&A] [answered]
Referring to the portion of the project where we define the `get_borrowable_data()` function.  In the `aave_borrow.py` file, the parameters are written as e.g. `total_collateral_eth` whereas on the Aave documentation site it's written as `totalCollateralETH`.

Is there a difference, or are both one in the same?

For my learning.  

Thanks!

**Comment by davidyuill27** [comment, marked as answer]
They are the same, the only thing that matters is the order has to be the same.
total_collateral_eth is just using the naming conventions in Python (all lower case with _ between words)
Solidity uses CamelCase for contracts and lowerCamelCase for variables.

See: https://en.wikipedia.org/wiki/Naming_convention_(programming)

**Reply by PatrickAlphaC** [reply]
Great answer!

**Comment by AlbertoSinigaglia** [comment]
There is no connection between the tuple returned in solidity and the returning value in your python script... you can literally use any name for your variables, it's called unpacking (in python, in other language you can find this feature under something named like "destructuring")  and you can find a short explanation [here](https://stackabuse.com/unpacking-in-python-beyond-parallel-assignment/)

**Reply by PatrickAlphaC** [reply]
Also great answer!
